### ISSUE TITLE
P3 preview branch build is failing

### ISSUE THREAD

**Issue by web-padawan** [vaadin/base-starter-vue#13]
For some reason, UglifyJS is failing on Polymer source code:

```sh
$ yarn build
yarn run v1.9.4
$ cross-env NODE_ENV=production webpack --progress --hide-modules
Hash: 7fe83019d2cf69950d59
Version: webpack 3.12.0
Time: 2634ms
       Asset    Size  Chunks                    Chunk Names
    build.js  811 kB       0  [emitted]  [big]  main
build.js.map  980 kB       0  [emitted]         main

ERROR in build.js from UglifyJs
Unexpected token: name (dedupeId) [./node_modules/@polymer/polymer/lib/utils/mixin.js:13,0][build.js:102,4]
```

Makes sense to verify after fixing #9

**Comment by Dev1an** [comment]
This seems to be [fixed in the react project](https://github.com/vaadin/base-starter-react/pull/16) but I don't understand how the fix works. Can anyone explain how we can get the vue project built?

**Comment by web-padawan** [comment]
Fixed by #14 and is now working on `p3-preview` branch, to be merged to master soon.
